src/js/playlist/playlist.js

```diff
@@ -1,3 +1,5 @@
+import { getPreload } from '../utils/preload';
```

Comment:
put this module in playlist if that's the only place it's used. "utils" is a not a miscellaneous folder for javascript modules. It's main purpose is for code that implements helpers found in `jwplayer.utils`.

Modules that serve a single script should be placed next to that script or a sub directory.